lib/statement-builder/statements/create.js

```diff
@@ -1,56 +1,46 @@
-var util = require('util');
-var Statement = require('../statement');
+const util = require('util');
+const Statement = require('../statement');
 
-var CreateStatement = module.exports = function () {
-  Statement.apply(this, arguments);
-};
+class CreateStatement extends Statement {
+  _init(options, entity) {
+    const opts = {};
+    const ret = this.schema.validate(this.schema.fixKeys(entity), 'create');
+    opts.entity = this.schema.deNull(ret);
 
-util.inherits(CreateStatement, Statement);
+    //
+    // Allow ttl to be passed into an insert
+    //
+    if (options.ttl) opts.ttl = options.ttl;
 
-CreateStatement.prototype._init = function (options, entity) {
-  var opts = {};
-  var ret = this.schema.validate(this.schema.fixKeys(entity), 'create');
-  if (this.typeOf(ret) === 'error') {
-    return ret;
+    return opts;
   }
 
-  opts.entity = this.schema.deNull(ret);
-
-  if (this.typeOf(opts.entity) === 'error') {
-    return opts.entity;
+  build(options) {
+    const allFields = this.schema.fields();
+    const entity = options.entity;
+    //
+    // Handle lookup table writes.
+    //
+    const table = options.table || this.table;
+
+    this.cql = util.format(
```

Comment:
I am for replacing `util.format()` usages with template strings, though...

Reply:
@scommisso: done, except for one case where user supplied strings are processed by  `util.format`.